druid-shell/src/platform/windows/window.rs

```diff
@@ -159,6 +162,25 @@ pub struct WindowHandle {
     state: Weak<WindowState>,
 }
 
+#[cfg(feature = "raw-win-handle")]
+unsafe impl HasRawWindowHandle for WindowHandle {
+    fn raw_window_handle(&self) -> RawWindowHandle {
+        if let Some(hwnd) = self.get_hwnd() {
+            let handle = WindowsHandle {
+                hwnd: hwnd as *mut libc::c_void,
+                hinstance: unsafe {
+                    winapi::um::libloaderapi::GetModuleHandleW(0 as winapi::um::winnt::LPCWSTR)
+                        as *mut libc::c_void
+                },
+                ..WindowsHandle::empty()
+            };
+            RawWindowHandle::Windows(handle)
+        } else {
+            panic!("Cannot retrieved HWMD for window.");
+        }
```

Comment:
Isn’t it HWND

Reply:
I’m not convinced panicking is right for any of these. I think they should return an empty window handle and maybe log a warning. Given that the raw window handle can represent empty handles.

Reply:
If we're okay with the idea that a `RawWindowHandle` can potentially be empty then I agree, otherwise we could consider making this an `Option<RawWindowHandle>`.

Reply:
This is an external crate, that is depended on by various things like wgpu and video libraries (and some other gui/rendering libraries). The RawWindowHandles all explicitly have an empty state and consumers have to handle that.

Reply:
Right, typo.

Reply:
I think a `panic!` is justified because if we have a `WindowHandle` without a native `HWND` I'm assuming we're in serious trouble. Or is there a valid state where that happens?

Reply:
It might be possible when the window is in the process of being set up or torn down? Elsewhere in the windows backend we log when we can't get the `HWND`, so I might just keep doing that. 🤷

Reply:
I think it is better to log. These handles are meant to represent empty states and users of them are expected to handle those states (its better to have an error on screen than a program crash!)